repose-aggregator/tests/functional-tests/build.gradle

```diff
@@ -1,3 +1,6 @@
+evaluationDependsOn(':repose-aggregator:tests:test-containers:test-container-glassfish')
+evaluationDependsOn(':repose-aggregator:tests:test-containers:test-container-tomcat')
```

Comment:
This may not be the cleanest way to do this. We could probably throw a bit of time at seeing if there's a better way of enforcing this.

For multi-project Gradle builds, "The default evaluation order of projects is alphanumeric (for the same nesting level)." [1] As a result, this Gradle build file runs before these two other build files. The copyArtifacts task
requires these other two build files to run first, as they declare configurations that are expected to be available..

[1] https://docs.gradle.org/current/userguide/multi_project_builds.html#sec:dependencies_which_dependencies